.gitignore

```diff
@@ -30,3 +30,6 @@ lerna-debug.log
 
 # Yeoman file
 .yo-rc.json
+
+# Yarn lock file
+yarn.lock
```

Comment:
This has been already merged in #455. Please rebase.

Reply:
i rebased =/

Reply:
I'll try to see what i can get with this... rebasing again...

Reply:
Rebased from `next`?

Reply:
Yes, when i do git rebase upstream/next it says it has nothing to update. Neither a diff shows any difference. I've also copied next to my fork and it says the same.

Reply:
@ematipico @dhruvdutt reverted everything all the way from the rebase, where my branch started. Now git managed to do the rebase correctly and i only replayed all the commands...